File: examples/example4-combo/src/main/java/mvu/sample/MainUI.java
package mvu.sample;

import com.vaadin.server.VaadinRequest;
import com.vaadin.ui.Component;
import com.vaadin.ui.VerticalLayout;

public class MainUI extends com.vaadin.ui.UI {
	@Override
	public void init(VaadinRequest request) {
		VerticalLayout layout = new VerticalLayout();

		Component mainComponent = Main.view();
		layout.addComponent(mainComponent);

		setContent(layout);
	}
}

File: examples/example5-simple-counter-kt/src/main/kotlin/mvu/sample/MainUI.kt
package mvu.sample

import com.vaadin.server.VaadinRequest
import com.vaadin.ui.VerticalLayout

class MainUI : com.vaadin.ui.UI() {
    public override fun init(request: VaadinRequest) {
        val layout = VerticalLayout()

        val mainComponent = Main.view()
        layout.addComponent(mainComponent)

        content = layout
    }
}
